frontend/src/app/modules/auth/login-page/login-page.component.html

```diff
@@ -32,16 +33,13 @@ <h2>Sign in to Recodo</h2>
                     <button class="login-with-button">
                         <img src="../../../../assets/icons/googleButton.svg" alt="google">
                     </button>
-                    <button class="login-with-button">
```

Comment:
Now, this space doesn't look very sexy
![image](https://user-images.githubusercontent.com/20342130/154925040-1a85764d-c0ef-414a-8f66-e2aae605dcf2.png)

Reply:
What's wrong? Or what do you want?

Reply:
Ok
If you two think the current implementation looks nice, I don't want to be a pain in the ass